tests/test_commands.py

```diff
@@ -188,6 +188,10 @@ def test_same_name_as_project(self):
         self.assertEqual(2, self.call('genspider', self.project_name))
         assert not exists(join(self.proj_mod_path, 'spiders', '%s.py' % self.project_name))
 
+    def test_same_name_as_existing_spider(self):
+        self.call('genspider', 'example', 'example.com')
+        self.assertEqual(0, self.call('genspider', 'example', 'example.com'))
```

Comment:
This only checks that the command finishes correctly with an exit code 0. Is there a way we could also test that the file is not overridden? Checking that the printed message appears in the command's output should be enough I think.

Reply:
I think we could maybe compare the md5 checksum of the file before and after if it's really necessary. Do you think we need that or should I just stick to checking the printed message?

Reply:
Just checking the message would be enough IMHO. I see you already made the change, thanks for that. Could you also check that the message does not appear the first time?

Reply:
Yep sure, will add it to the test

Reply:
Have added the check